src/main/java/be/syntra/devshop/DevshopBack/services/DataFillerServiceImpl.java

```diff
@@ -180,6 +185,35 @@ public void initialize() {
                             "admin@emaill.com")
             ));
         }
+
+        if (cartRepository.count() == 0) {
+            userRepository.findAll().forEach(this::setArchivedCartsAndSave);
+            userRepository.flush();
```

Comment:
https://stackoverflow.com/questions/32108491/what-is-exact-purpose-of-flush-in-jpa

Reply:
the first line is : "In theory, you (as a user of JPA) should never (or in absolutely rare situations) get in a situation to call flush()." hahaha, removed it, still works.